tests/main.test.ts

```diff
@@ -1,22 +1,323 @@
 import { db } from "./__mocks__/db";
 import { server } from "./__mocks__/node";
 import usersGet from "./__mocks__/users-get.json";
-import { expect, describe, beforeAll, beforeEach, afterAll, afterEach, it } from "@jest/globals";
+import { expect, describe, beforeAll, beforeEach, afterAll, afterEach, it, jest } from "@jest/globals";
+import { Logs } from "@ubiquity-dao/ubiquibot-logger";
+import { Context, SupportedEventsU } from "../src/types";
+import { drop } from "@mswjs/data";
+import issueTemplate from "./__mocks__/issue-template";
+import repoTemplate from "./__mocks__/repo-template";
+import { askQuestion } from "../src/handlers/ask-gpt";
+import { plugin } from "../src/plugin";
 
-beforeAll(() => server.listen());
-afterEach(() => server.resetHandlers());
+const TEST_QUESTION = "What is pi?";
+const TEST_SLASH_COMMAND = "/gpt what is pi?";
+const LOG_CALLER = "_Logs.<anonymous>";
+
+type Comment = {
+  id: number;
+  user: {
+    login: string;
+    type: string;
+  };
+  body: string;
+  url: string;
+  html_url: string;
+  owner: string;
+  repo: string;
+  issue_number: number;
+  issue_url?: string;
+  pull_request_url?: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const octokit = jest.requireActual("@octokit/rest") as any;
```

Comment:
Why not just use the correct object? Why cast as `any`?

Reply:
the `jest` import from `@jest/globals` isn't the same as the namespace through `@types/jest` and types as unknown